src/mpyl/steps/deploy/k8s/helm.py

```diff
@@ -68,7 +51,7 @@ def install(logger: Logger, step_input: Input, name_space: str, kube_context: st
 
     if step_input.dry_run:
         cmd = f"helm upgrade -i {chart_name} -n namespace --kube-context {kube_context} {chart_path} --debug --dry-run"
-        return custom_check_output(logger, cmd)
+        return custom_check_output(logger, cmd.split(" "))
 
     cmd = f"helm upgrade -i {chart_name} -n {name_space} --kube-context {kube_context} {chart_path}"
-    return custom_check_output(logger, cmd)
+    return custom_check_output(logger, cmd.split(" "))
```

Comment:
Why do you want to do the command splitting on the outside?

Reply:
In sbt build's case I have to supply the DOCKER_BUILDKIT env var as well, which made it a bit cumbersome to do with splitting by default.. But I can take a look to improve this

Reply:
Ah, right. I see. 🤔 maybe we could support two alternative apis? Or use a union type that accepts a string or a list of strings.